README.md

```diff
@@ -3,18 +3,43 @@ NEAR Crossword
 
 How to play with this contract
 ===============================
-1. Clone the repo and build it with `yarn build`
-2. Run `near dev-deploy` to deploy the contract to `testnet`.
-3. Create a crossword, let's say that the answer to your crossword is "many clever words"
-4. Answer for your crossword from now on will be a seed phrase! Let's generate key pair out of it.
+1. Clone the repo.
+
+```
+git clone https://github.com/near-examples/near-crossword.git
+cd near-crossword
+```
+
+2. Next, make sure you have NEAR CLI by running:
+
+  ```
+  near --version
+  ```
+
+  If you need to install `near-cli`:
+
+  ```
+  npm install near-cli -g
+  ```
+
+3. Build the smart contract
+
+```
+cd contract
+./build.sh
+```
+
+3. Run `near dev-deploy` to deploy the contract to `testnet`.
```

Comment:
Looks like we have `3.` twice in this bullet list

Reply:
thanks, addressed in latest commit